File: TI1.6 SOAP_DOTNET_CONBDD_EUREKABANK_GR03/01. SERVIDOR/ec.edu.monster.modelo/Cuenta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace ec.edu.monster.modelo
{
    [DataContract]
    public class Cuenta
    {
        [DataMember]
        public string NumeroCuenta { get; set; }

        [DataMember]
        public string NombreCliente { get; set; }

        [DataMember]
        public double Saldo { get; set; }

        [DataMember]
        public string Moneda { get; set; }

        [DataMember]
        public string Estado { get; set; }

        public Cuenta() { }

        public Cuenta(string numeroCuenta, string nombreCliente, double saldo, string moneda, string estado)
        {
            NumeroCuenta = numeroCuenta;
            NombreCliente = nombreCliente;
            Saldo = saldo;
            Moneda = moneda;
            Estado = estado;
        }
    }
}

File: TI1.7 REST_JAVA_CONDBB_EUREKABANK_GR03/02. CLIMOV/app/src/main/java/ec/edu/espe/rest_java_bank/data/models/Cuenta.kt
package ec.edu.espe.rest_java_bank.data.models

import com.google.gson.annotations.SerializedName

data class Cuenta(
    @SerializedName("numeroCuenta")
    val numeroCuenta: String,
    @SerializedName("nombreCliente")
    val nombreCliente: String,
    @SerializedName("saldo")
    val saldo: Double,
    @SerializedName("moneda")
    val moneda: String,
    @SerializedName("estado")
    val estado: String
)
